src/ReinforcementLearningBase/src/base.jl

```diff
@@ -104,8 +104,8 @@ function test_interfaces!(env)
             A, A′ = legal_action_space(X), legal_action_space(Y)
```

Comment:
I know this is unrelated to your PR but shouldn't these tests be in the test directory? Can you perhaps copy paste them in a new file there, otherwise I suspect they'll stay here a while longer.

Reply:
The `test_` functions are part of base and let you test your environments. So they aren't tests, but rather 'test mechanisms', which we use in the tests (but other people can / do as well).